src/styles/homeForm.scss

```diff
@@ -0,0 +1,32 @@
+$backColor: #0f0f0f;
+$baseColor: #FFFFFF;
+
+.header {
+  position: fixed;
+  width: 100%;
+  
+}
+
+.site-layout {
+    background: #fff;
+    padding: 0 50px !important;
+    margin-top: 64px !important;
+}
+
+.breadcrumb{
+    margin: 20px 0 !important;
```

Comment:
this important statements are really needed? or we can use multiple class selector to avoid importants?